src/adapters/refract-adapter.coffee

```diff
@@ -0,0 +1,45 @@
+_ = require('./refract/helper')
+blueprintApi = require('../blueprint-api')
+
+getDescription = require('./refract/getDescription')
+transformSections = require('./refract/transformSections')
+
+transformAst = (element) ->
+
+  applicationAst = new blueprintApi.Blueprint({
+    name: _.chain(element).get('meta.title', '').fixNewLines().value()
+    version: '22' # how about this? we don't have in refract parse result at all
+    metadata: []
+  })
+
+  # Metadata and location
+  applicationAst.metadata =
+    _.chain(element)
+    .get('attributes.meta')
+    .filter({meta: {classes: ['user']}})
+    .map((entry) ->
+      content = _.content(entry)
+
+      name = _(entry).content().get('key.content')
+      value = _(entry).content().get('value.content', '')
```

Comment:
A bit of inconsistency here, sometimes we use the `content` function, some times we access it using `.get(something.content)`, but that's a niptick (and probably I am the author of the code as well)